osclients.py

```diff
@@ -31,12 +31,24 @@
 from keystoneclient.v2_0 import client as keystonev2
 from keystoneclient.v3 import client as keystonev3
 
-from neutronclient.v2_0 import client as neutronclient
-from novaclient import client as novaclient
-from cinderclient.v2 import client as cinderclient
-from glanceclient import client as glanceclient
-from swiftclient import client as swiftclient
-
+if 'OSCLIENTS_MODULES' not in env:
```

Comment:
[Is it pythonic to import inside functions?] (http://stackoverflow.com/questions/1024049/is-it-pythonic-to-import-inside-functions)